airbyte-webapp-e2e-tests/cypress/commands/connection.ts

```diff
@@ -8,7 +8,7 @@ export const createTestConnection = (sourceName: string, destinationName: string
 
   createTestSource(sourceName);
   createTestDestination(destinationName);
-  cy.wait(3000);
+  cy.wait(5000);
```

Comment:
Now that we're calling an API instead of a local database query, this test needed a little more time to complete

Reply:
I was indeed seeing this fail. These tests are loading a real browser and clicking around, so there has to be some built in wait time between when the test clicks 'create' and when the test can check for the rendered result